### DISCUSSION TITLE
Rate limiting for Kafka consumer | kafka消费速率控制-sentinel

### DISCUSSION THREAD

**Discussion by martixZero** [alibaba/Sentinel#2359] [Ideas]
想问下，如果在kafka消费端，想使用sentinel进行kafka消费速率控制，避免消费速率过快，影响下游依赖服务，可以使用sentinel或者更好的解决方案么？辛苦盼回复

**Comment by sczyh30** [comment]
可以的，在消息消费的地方进行埋点，然后配置匀速排队模式的流控规则即可。
